prototype_source/quantization_in_pytorch_2_0_export_tutorial.rst

```diff
@@ -0,0 +1,630 @@
+            )
+            act_qspec = act_quantization_spec
+            weight_qspec = weight_quantization_spec
+
+            def derive_qparams_fn(obs_or_fqs: List[ObserverOrFakeQuantize]) -> Tuple[Tensor, Tensor]:
+                assert len(obs_or_fqs) == 2, \
+                    "Expecting two obs/fqs, one for activation and one for weight, got: {}".format(len(obs_or_fq))
+                act_obs_or_fq = obs_or_fqs[0]
+                weight_obs_or_fq = obs_or_fqs[1]
+                act_scale, act_zp = act_obs_or_fq.calculate_qparams()
+                weight_scale, weight_zp = weight_obs_or_fq.calculate_qparams()
+                return torch.tensor([act_scale * weight_scale]).to(torch.float32), torch.tensor([0]).to(torch.int32)
+
+            bias_qspec = DerivedQuantizationSpec(
+                derived_from=[(input_act, node), (weight, node)],
+                derive_qparams_fn=derive_qparams_fn,
+                dtype=torch.int32,
+                quant_min=-2**31,
+                quant_max=2**31 - 1,
+                qscheme=torch.per_tensor_symmetric,
+            )
+            input_qspec_map = {input_act: act_qspec, weight: weight_qspec, bias: bias_qspec}
+            node.meta["quantization_annotation"] = QuantizationAnnotation(
+                input_qspec_map=input_qspec_map,
+                output_qspec=act_qspec,
+                _annotated=True,
+            )
+
+5. A Toy Example with Resnet18 
+--------------------------------------------------------
+
+After above annotation methods defined with ``QuantizationAnnotation API``, we can now put them together to construct a ``BackendQuantizer``
+to run a example with Torchvision Resnet18. Here are some basic concepts before we move on to this example:
+
+- `QuantizationSpec <https://github.com/pytorch/pytorch/blob/73fd7235ad25ff061c087fa4bafc6e8df4d9c299/torch/ao/quantization/_pt2e/quantizer/quantizer.py#L28-L66>`__
+  defines the ``data type``, ``qscheme``, and other quantization parameters used to quantize a tensor.
+- `QuantizationConfig <https://github.com/pytorch/pytorch/blob/73fd7235ad25ff061c087fa4bafc6e8df4d9c299/torch/ao/quantization/_pt2e/quantizer/quantizer.py#L103-L109>`__
+  consists of ``QuantizationSpec`` for activation, weight, and bias separately.
+- When annotating the model, methods of
+  `get_act_qspec <https://github.com/pytorch/pytorch/blob/73fd7235ad25ff061c087fa4bafc6e8df4d9c299/torch/ao/quantization/_pt2e/quantizer/utils.py#L9>`__,
+  `get_weight_qspec <https://github.com/pytorch/pytorch/blob/73fd7235ad25ff061c087fa4bafc6e8df4d9c299/torch/ao/quantization/_pt2e/quantizer/utils.py#L26>`__, and
+  `get_bias_qspec <https://github.com/pytorch/pytorch/blob/73fd7235ad25ff061c087fa4bafc6e8df4d9c299/torch/ao/quantization/_pt2e/quantizer/utils.py#LL42C5-L42C19>`__
+  can be used to get the ``QuantizationSpec`` from ``QuantizationConfig`` for a specific node.
+
+.. code:: ipython3
```

Comment:
please remove this code as well

Reply:
Thanks this code are removed and add a gist link here.